hack/verify-go.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+go test ./...
```

Comment:
nit: you're not guaranteeing you're at repo root when you run this

One option is to hardcode the relative location of repo root to the script, eg:
```sh
REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
cd "${REPO_ROOT}"
```

Another option is to avoid the hardcode, but assume you're in a git repo, eg:
```sh
REPO_ROOT=$(git rev-parse --show-toplevel)
cd "${REPO_ROOT}"
```